packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -156,6 +159,38 @@ class SemanticsData extends Diagnosticable {
   /// if this node represents a text field.
   final TextSelection textSelection;
 
+  /// Indicates the current scrolling position in logical pixels if the node is
+  /// scrollable.
+  ///
+  /// The properties [scrollExtentMin] and [scrollExtentMax] indicate the valid
+  /// in-range values for this property. The value for [scrollPosition] may
+  /// (temporarily) be outside that range, e.g. during an overscroll.
+  ///
+  /// See also:
+  ///
+  ///  * [ScrollPosition.pixels] from where this value is usually taken.
```

Comment:
here and at least 5 other cases: put a comma after the member name, so it reads "foo, from where bar", rather than "foo from where bar".